src/utils/isBrowser.ts

```diff
@@ -1,8 +1,2 @@
-/**
- * Though the process variable can be redefined, this approach is suitable for now.
- * 
- * Another option is to use new Function() constructor so the execution scope of this is binded to the global scope.
- * However, this fails if Content-Security-Policy (CSP) script-src does not allow 'unsafe-eval'.
- */
 export const isBrowser = () =>
-  typeof process === "undefined" || process.release?.name.search(/node|io.js/) === -1;
+  typeof process !== "undefined" && process.release?.name.search(/node|io.js/) !== -1;
```

Comment:
What is wrong with this approach?

Reply:
@JGiter In the Switchboard app, the `process` variable is defined, but `process.release` is not, so the new `isBrowser` is returning `false` in Switchboard 🤦
![image](https://user-images.githubusercontent.com/4407464/103795423-2fae2d00-5003-11eb-9530-ca3fbce02cbb.png)